brian2/codegen/translation.py

```diff
@@ -297,10 +297,6 @@ def make_statements(code, variables, dtype, optimise=True, blockname=''):
         elif not variables[stmt.var].scalar:
             scalar_write_done = True
 
-    # all variables which are written to at some point in the code block
-    # used to determine whether they should be const or not
-    all_write = set(line.write for line in lines)
-
```

Comment:
Is there a case for leaving this here purely as a form of documentation?

Reply:
I'd find it a bit confusing, especially since it says that it uses this to determine the `const` modifier which is not true (it uses `will_write` for that).

Reply:
Fair enough! :-)